src/main/java/duke/UI.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.util.Scanner;
+import java.lang.StringBuilder;
+
+
+/**
+ * This class is in charge of the user interface.
+ *
+ * It is in charge responding to the user.
+ */
+public class UI {
```

Comment:
Perhaps you should use the name Ui instead of UI, to match the tendency to associate abbreviated names as 1 word?